README.md

```diff
@@ -323,7 +306,7 @@ Boolean, set to true to log LDAP auth.
 
 ####`manage_repos`
 
-Boolean, whether or not to manage package repositories.
+This option is deprecated in favor of repos_ensure, and is no longer honored.
```

Comment:
Got it. I think @DavidS had asked me to put warnings in the code; do you want those still left in (if you try to call the deprecated parameters)?